src/index.js

```diff
@@ -1,21 +1 @@
-import _ from 'lodash';
-import './style.css';
-import Icon from './icon.png';
-
-function component() {
-  const element = document.createElement('div');
-
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
-
-  // Add the image to our existing div.
-  const myIcon = new Image();
-  myIcon.src = Icon;
-
-  element.appendChild(myIcon);
-
-  return element;
-}
-
-document.body.appendChild(component());
```

Comment:
Great job clearing the index.js file from having the setup stage codes. Please also consider clearing the lodash dependency from the package.json file.